modules/client_options/options.lua

```diff
@@ -153,6 +149,12 @@ local function setupComboBox()
         end
     end
 
+    for _, preset in ipairs(Keybind.presets) do
+        listKeybindsPanel:addOption(preset)
+    end
+    listKeybindsPanel.onOptionChange = function(comboBox, option)
+        setOption('listKeybindsPanel', option)
+    end
```

Comment:
I spent a good while thinking about why it wasn't working; when opening the client, it doesn't load the last saved keybinds.

```suggestion
    for _, preset in ipairs(Keybind.presets) do
        listKeybindsPanel:addOption(preset)
    end
    listKeybindsPanel.onOptionChange = function(comboBox, option)
        setOption('listKeybindsPanel', option)
    end
    panels.keybindsPanel.presets.list:setCurrentOption(Keybind.currentPreset)
```